geniza/corpus/templates/admin/corpus/document/rev_log_entry.html

```diff
@@ -0,0 +1,4 @@
+<div class="readonly">
+<time>{% if log_entry.legacy_date %}{{ log_entry.legacy_date }}{% else %}{{ log_entry.action_time|date:"DATETIME_FORMAT" }}{% endif %}</time>
```

Comment:
For first input can we just display date without time? Maybe a flag to the template to specify not to show time. (Asking because it's odd to see the `midnight` in the historic info)

Reply:
Makes sense. We don't have that info for the entire corpus now, and even for newly added documents it probably doesn't matter as much as the date

Reply:
Switched to `naturaldate` for both, since:
- for old dates (i.e. most "first input" dates) it looks the same
- for newer dates, it's easier to know how recently something was modified
- it seems unlikely that records would have many edits per day, so the timestamp isn't as important
- if you need the timestamp, you can view the full history

lmk if this reasoning makes sense to you; we could also ask the team today.

Reply:
naturaldate is a great solution